Header header
string[] joint_names
trajectory_msgs/JointTrajectoryPoint[] points
---
Header header
string[] joint_names
trajectory_msgs/JointTrajectoryPoint[] points
